rules/cross-platform/defense_evasion_timestomp_touch.toml

```diff
@@ -24,9 +27,9 @@ type = "eql"
 
 query = '''
 process where event.type == "start" and
- process.name : "touch" and user.id != "0" and
- process.args : ("-r", "-t", "-a*","-m*") and
- not process.args : ("/usr/lib/go-*/bin/go", "/usr/lib/dracut/dracut-functions.sh", "/tmp/KSInstallAction.*/m/.patch/*")
+ (process.name : "touch" and user.id != "0" and process.args : ("-r", "-t", "-a*","-m*")) or 
+ (process.name : "touch" and process.args : ("-acmr", "/bin/*", "/opt/*")) and
```

Comment:
`(process.name : "touch" and process.args : ("-acmr", "/bin/*", "/opt/*")`

This works as an OR condition, and not an AND condition, this means basically the following:

`process.name:"touch" and process.args:("-acmr" OR "/bin/*" OR "/opt/*")`